app/src/main/java/com/compscieddy/meetinthemiddle/activity/GroupActivity.java

```diff
@@ -693,16 +738,38 @@ public void run() {
     });
   }
 
-  private void resizeViewPager(boolean isFABClick) {
+  public void resizeViewPager(boolean isFABClick) {
     Display display = getWindowManager().getDefaultDisplay();
     Point size = new Point();
     display.getSize(size);
     int height = size.y;
 
     ResizeAnimation resizeAnimation;
-
     if (isFABClick) {
       if (isViewPagerCollapsed) {
+        resizeAnimation = new ResizeAnimation(
+            mBottomSection,
+            (int) (height * CHAT_EXPANDED_HEIGHT),
+            (int) (height * CHAT_MINIMIZED_HEIGHT));
+        isViewPagerCollapsed = !isViewPagerCollapsed;
```

Comment:
You can move these 3 lines out of the if/else statement. Because they are **exactly** identical to the lower lines